yb-voyager/src/utils/sqlname/sqlname.go

```diff
@@ -239,7 +240,11 @@ func IsAllUppercase(s string) bool {
 
 func IsAllLowercase(s string) bool {
 	for _, c := range s {
-		if c >= 'A' && c <= 'Z' {
+		if c == '_' {
+			continue
+		}
+		if (c >= 'A' && c <= 'Z') ||
```

Comment:
The OR is a bit confusing IMO. 
How about we check if above this line if `IsLetter` or `isDigit` and then just continue?